client/dashboard/app/queries/domain-dns-records.ts

```diff
@@ -42,3 +43,13 @@ export const domainDnsEmailMutation = ( domainName: string ) =>
 			} );
 		},
 	} );
+
+export const domainDnsImportBindMutation = ( domainName: string ) =>
+	mutationOptions( {
+		mutationFn: ( file: File ) => importDnsBind( domainName, file ),
+		onSuccess: () => {
+			queryClient.invalidateQueries( {
+				queryKey: [ 'domains', domainName, 'dns' ],
+			} );
```

Comment:
We can reuse the queries 😄 


```suggestion
			queryClient.invalidateQueries( domainDnsQuery( domainName ) );
```